Octokit.Tests/Models/CheckRunEventTests.cs

```diff
@@ -256,5 +256,257 @@ public void CanBeDeserialized()
             Assert.Equal(CheckStatus.Completed, payload.CheckRun.CheckSuite.Status.Value);
             Assert.Equal(CheckConclusion.Neutral, payload.CheckRun.CheckSuite.Conclusion);
         }
+
+        [Fact]
+        public void CanBeDeserializedWithEnterpriseAppOwner() {
+            const string json = @"{
+  ""action"": ""rerequested"",
+  ""check_run"": {
+    ""id"": 4,
+    ""head_sha"": ""d6fde92930d4715a2b49857d24b940956b26d2d3"",
+    ""external_id"": """",
+    ""url"": ""https://api.github.com/repos/github/hello-world/check-runs/4"",
+    ""html_url"": ""http://github.com/github/hello-world/runs/4"",
+    ""status"": ""completed"",
+    ""conclusion"": ""neutral"",
+    ""started_at"": ""2018-05-04T01:14:52Z"",
+    ""completed_at"": ""2018-05-04T01:14:52Z"",
+    ""output"": {
+      ""title"": ""Report"",
+      ""summary"": ""It's all good."",
+      ""text"": ""Minus odio facilis repudiandae. Soluta odit aut amet magni nobis. Et voluptatibus ex dolorem et eum."",
+      ""annotations_count"": 2,
+      ""annotations_url"": ""https://api.github.com/repos/github/hello-world/check-runs/4/annotations""
+    },
+    ""name"": ""randscape"",
+    ""check_suite"": {
+      ""id"": 5,
+      ""head_branch"": ""main"",
+      ""head_sha"": ""d6fde92930d4715a2b49857d24b940956b26d2d3"",
+      ""status"": ""completed"",
+      ""conclusion"": ""neutral"",
+      ""url"": ""https://api.github.com/repos/github/hello-world/check-suites/5"",
+      ""before"": ""146e867f55c26428e5f9fade55a9bbf5e95a7912"",
+      ""after"": ""d6fde92930d4715a2b49857d24b940956b26d2d3"",
+      ""pull_requests"": [
+
+      ],
+      ""app"": {
+        ""id"": 2,
+        ""node_id"": ""MDExOkludGVncmF0aW9uMQ=="",
+        ""owner"": {
+          ""login"": ""github"",
+          ""id"": 340,
+          ""node_id"": ""MDEyOk9yZ2FuaXphdGlvbjE="",
+          ""avatar_url"": ""http://alambic.github.com/avatars/u/340?"",
+          ""gravatar_id"": """",
+          ""url"": """",
+          ""html_url"": """",
+          ""followers_url"": """",
+          ""following_url"": """",
+          ""gists_url"": """",
+          ""starred_url"": """",
+          ""subscriptions_url"": """",
+          ""organizations_url"": """",
+          ""repos_url"": """",
+          ""events_url"": """",
+          ""received_events_url"": """",
+          ""type"": ""enterprise"",
+          ""site_admin"": false
+        },
```

Comment:
This is the same payload as the test above, except with the two `app.owner` entries modified to look like an enterprise owner.